src/js/components/Home.js

```diff
@@ -1,27 +1,21 @@
-import { select, templates } from '../settings.js';
+import { templates } from '../settings.js';
+import CarouselWidget from '../components/CarouselWidget.js';
 
 class Home {
-  constructor(element){
+  constructor(element) {
     const thisHome = this;
     thisHome.render(element);
-    thisHome.initWidgets();
+    CarouselWidget.initWidgets();
```

Comment:
Ten kod nie zadziała. Widać, że go nie odpaliłeś w ogóle. 
To co tutaj zrobiłeś to jest statyczne wywołanie metody, gdzie metoda `initWidgets` nie jest metodą statyczną (poczytaj o metodach statycznych w js).

W konstruktorze klasy już tę metodę wywołujesz, więc zasadniczo wystarczy stworzyć instancję klasy `CarouselWidget` i nic więcej nie potrzeba. 

Do poczytania: https://kursjs.pl/kurs/obiekty/class